lib/mix/tasks/firmware.ex

```diff
@@ -314,8 +314,25 @@ defmodule Mix.Tasks.Firmware do
 
   defp config_arg(:fwup_conf, firmware_config) do
     case firmware_config[:fwup_conf] do
-      nil -> []
-      fwup_conf -> ["-c", Path.join(File.cwd!(), fwup_conf)]
+      nil ->
+        []
+
+      fwup_conf ->
+        path =
+          cond do
+            File.exists?(fwup_conf) ->
```

Comment:
It seems like `Path.expand/1` should be used here since it handles both absolute and relative paths.